stable/redis-ha/values.yaml

```diff
@@ -50,14 +50,15 @@ haproxy:
     annotations: {}
   serviceAccount:
     create: true
-  ## Prometheus metric exporter for HAProxy.
+  ## Official HAProxy embedded prometheus metrics settings.
+  ## Ref: https://github.com/haproxy/haproxy/tree/master/contrib/prometheus-exporter
   ##
-  exporter:
-    image:
-      repository: quay.io/prometheus/haproxy-exporter
-      tag: v0.9.0
+  metrics:
```

Comment:
@VLZZZ  @zanhsieh This is a breaking change and would require bumping the major version along with a snippet in the README.md on this and how to upgrade from a 3.x version to a 4.x version. @DandyDeveloper Could you also validate the upgrade in your cluster?

Reply:
Bumped major version and added upgrade notes to README.md.
As an option we may also configure deprecations template which will prevent chart from being deployed until `haproxy.exporter` presented in values.